charts/prometheus-postgres-exporter/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
```

Comment:
Not an issue but an FYI since we do not have pod security policies enabled at the cluster level this fill not have an effect.